third_party/nvidia/backend/driver.c

```diff
@@ -161,6 +170,27 @@ typedef CUresult (*cuTensorMapEncodeTiled_t)(
     CUtensorMapSwizzle swizzle, CUtensorMapL2promotion l2Promotion,
     CUtensorMapFloatOOBfill oobFill);
 
+#ifdef WIN32
+#define defineGetFunctionHandle(name, symbolName)                              \
+  static symbolName##_t name() {                                               \
+    /* Open the shared library */                                              \
+    HMODULE handle = LoadLibraryA("nvcuda.dll");                               \
+    if (!handle) {                                                             \
+      PyErr_SetString(PyExc_RuntimeError, "Failed to open nvcuda.dll");        \
+      return NULL;                                                             \
+    }                                                                          \
+    symbolName##_t funcHandle =                                                \
```

Comment:
Should we explicitly clear any existing error on Windows as well?
```suggestion
    }                                                                          \
    /* Clear any existing error */                                             \
    SetLastError(NO_ERROR);                                                    \
    symbolName##_t funcHandle =                                                \
```

Reply:
I don't think that is needed. GetLastError is used for information purposes only (think of it as of `errno` in POSIX), any next call of Win32 API should set its own error status, clear it if it is successful.